client/src/components/inventory/RequestButton.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import {Button, Icon} from "semantic-ui-react";
+import {useMutation, useQuery} from "@apollo/react-hooks";
+import {CREATE_REQUEST} from "../util/graphql/Mutations";
+import {RequestedItem} from "../../types/Hardware";
+import {GET_USER} from "../util/graphql/Queries";
+
+interface RequestButtonProps {
+    request: RequestedItem,
+}
+
+function RequestButton({request}: RequestButtonProps) {
+    const {subscribeToMore, ...query} = useQuery(GET_USER);
```

Comment:
No need to request current info about user this way.  You can get it through redux easily; see the bottom of HardwareList.tsx for an example